profile.d/toolbox.sh

```diff
@@ -10,7 +10,11 @@ toolbox_welcome_stub="$toolbox_config/toolbox-welcome-shown"
 # shellcheck disable=2046
 # shellcheck disable=SC1091
 eval $(
-          . /usr/lib/os-release
+          if [ -f /usr/lib/os-release ]; then
+              . /usr/lib/os-release
+          else
+              . /etc/os-release
```

Comment:
Strictly speaking, we should check `/etc/os-release` first and then fall back to `/usr/lib/os-release`.  The `os-release(5)` [manual](https://www.freedesktop.org/software/systemd/man/os-release.html) says:
```
The file /etc/os-release takes precedence over /usr/lib/os-release.
Applications should check for the former, and exclusively use its data
if it exists, and only fall back to /usr/lib/os-release if it is missing.
```

That's also what we do in `test/system/libs/helpers.bash`.